gradle.properties

```diff
@@ -25,4 +25,4 @@ org.gradle.jvmargs=-Xmx4g -Xms500m
 nodeBinaries.commit=8755ae4c05fd476cd23f2972049111ba436c86d4
 nodeBinaries.version=v20.12.2
 cody.autocomplete.enableFormatting=true
-cody.commit=6ac4a8c1831ad3945fc16f20b8f21947897d3d14
```

Comment:
- [ ] wait for https://github.com/sourcegraph/cody/pull/6144 and and for the release including the change